rotkehlchen/tests/fixtures/accounting.py

```diff
@@ -46,7 +46,7 @@ def fixture_data_dir(use_clean_caching_directory, tmpdir_factory) -> Path:
     # cached price queries, not for user DBs
     for x in data_directory.iterdir():
         if x.is_dir() and (x / 'rotkehlchen.db').exists():
-            shutil.rmtree(x)
+            shutil.rmtree(x, ignore_errors=True)
```

Comment:
what's this for?

Reply:
There is the case the accounting tests where not all the files in the user folder get deleted and this makes sure that all the folders get deleted even if there are files in them

Reply:
Thanks. I see. I guess something happened to lead you to do that.